src/pedal/mod.rs

```diff
@@ -0,0 +1,26 @@
+mod transparent;
```

Comment:
This tells the compiler that the `pedal` mod contains a sub-mod named `transparent`, which will exist either at `pedal/transparent.rs` or `pedal/transparent/mod.rs`. If it was `pub mod transparent` instead it would be publicly accessible, but instead it'll only be available here. (But I re-exported the `Transparent` struct down below publicly.)